### PR TITLE
Stefanchak_V | Implemented JDBC connection

### PR DESCRIPTION
**Pull request by SteVich**

### CODE REVIEW

File: src/main/java/com/academy/task/service/EmailService.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        return emailRepository.getById(id);
+    }
+
+    public void deleteEmail(Long id) {
+        emailRepository.deleteSent(id);
+    }
+
+    public Long getLargestId() {
+        List<Email> emails = emailRepository.getAll();
+        Optional<Email> resultList = emails.stream()
```
Comment (dgray16) [inline src/main/java/com/academy/task/service/EmailService.java]: Better to use SQL aggregation.

File: src/main/java/com/academy/task/util/ConnectionProperties.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@ConfigurationProperties(prefix = "db")
+public class ConnectionProperties {
+
+    String driver;
+
+    String URL;
+
+    String USER;
+
+    String PASS;
```
Comment (dgray16) [inline src/main/java/com/academy/task/util/ConnectionProperties.java]: Java naming convention states that non constant field of class should be named in camel case.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,16 @@
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=stevi.inventorsoftacademy@gmail.com
+spring.mail.password=inventorsoft
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.smtp.starttls.enable=true
+
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL95Dialect
```
Comment (dgray16) [inline src/main/resources/application.properties]: In your code these properties do not work.

File: src/main/java/com/academy/task/controller/EmailController.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    @GetMapping("{id}")
+    public ResponseEntity<Email> findEmailById(@PathVariable Long id) {
+        return ResponseEntity.ok(emailService.getEmailById(id));
+    }
+
+    @PostMapping
+    public ResponseEntity<String> createEmail(@Valid @RequestBody Email email) {
+        emailService.addEmail(email);
+
+        return ResponseEntity.ok().body("Email created successfully and will be sent at " + email.getDate().toLocalTime());
```
[the remark this answers is not in the document]
Comment (SteVich) [reply src/main/java/com/academy/task/controller/EmailController.java]: I'm not sure what that means exactly .. Do I need to create something  Response object and return it to the body?

[the remark this answers is not in the document]
Comment (dgray16) [reply src/main/java/com/academy/task/controller/EmailController.java]: ResponseEntity#ok(T) accepts body as param.
No need to call explicitly body() method.

**Pull request closed without merging**